src/services/users/userService.ts

```diff
@@ -0,0 +1,76 @@
+import { v4 as uuid } from "uuid";
+import { Op, literal } from "sequelize";
+import sequelize from "../../data-acess/db";
+import { User as UserModel } from "../../models/User";
+import { User, UserInternalProps } from "../../types/user";
+
+export default class UserService {
+    async getUserById(id: string) {
+        await sequelize.sync();
+
+        return await UserModel.findByPk(id)
+    }
+
+    async createUser(user: Omit<User, UserInternalProps>) {
+        const { login, password, age } = user;
+        await sequelize.sync();
+
+        return await UserModel.create({
+            login,
+            password,
+            age,
+            id: uuid(),
+            isDeleted: false,
+        });
+    }
+
+    async updateUserById(id: string, user: Partial<Omit<User, UserInternalProps>>) {
+        const { login, password, age } = user;
+        await sequelize.sync();
+
+        const currentUser = await this.getUserById(id);
+
+        if (!currentUser) {
+            return;
+        }
+
+        await currentUser.update({
+            login,
+            password,
+            age,
+        });
+        await currentUser.save();
```

Comment:
method update already saves the data in db, so there is no sense to call save

Reply:
Fixed into https://github.com/SorokinTim/nodejs-gmp/pull/6/commits/a016fe24a80382fa01f5a13b7bf7a8b8ba161166 commit